src/App.tsx

```diff
@@ -10,33 +11,28 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabIdState, setSelectedTabIdState] = useState(tabs[0].id);
+  const [selectedTabTitle, setSelectedTabTitle] = useState(tabs[0].title);
+  const [selectedTabContent, setSelectedTabContent] = useState(tabs[0].content);
+  const changeTab = (tabId: string, tabTitle: string, tabContent: string) => {
+    setSelectedTabIdState(tabId);
+    setSelectedTabTitle(tabTitle);
+    setSelectedTabContent(tabContent);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {' '}
+        {selectedTabTitle}
```

Comment:
You can use [interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here. It will make your code cleaner